Makefile

```diff
@@ -75,6 +81,12 @@ container-scan: container
 container-public: container
 	docker push ${DOCKER_USER}/observability-stack:${DOCKER_TAG}
 
+checkout-cluster-monitor:
```

Comment:
It's all getting a bit messy in the `build` target with some copies there but then other targets like this that checkout directly. I don't really like it at all but it's a necessary evil, however we should be consistent I think.

I'd propose we have a separate target for each of the current copies we do, similar to this one. We then have a helper target `setup-container-source` or something that runs all of them which `build` and others can depend on.

Each of those targets can then do any preparation work to clean the destination - we should also have clean up for all of them at least in `clean` but again it might be better to have a matching add/remove target for each to keep things consistent and ensure we revert correctly.